Gsp-kubernetes-best-practices-guide.html

```
@@ -0,0 +1,1422 @@
+                .main {
+                    margin-left: 0;
+                    padding: 80px 24px 64px;
+                }
+                .menu-toggle {
+                    display: flex;
+                }
+            }
+        </style>
+    </head>
+    <body>
+        <button class="menu-toggle" id="menuToggle" aria-label="メニューを開く">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)